frontend/src/core/project/views/projectDetailView.js

```diff
@@ -60,6 +66,82 @@ define(function(require) {
 
       var messageText = typeof response.responseJSON == 'object' && response.responseJSON.message;
       EditorOriginView.prototype.onSaveError.call(this, null, messageText);
+    },
+
+    /**
+     * When a new course is created it gets populated with a page, article, block and blank component
+     * so that it can be previewed immediately.
+     * @param model
+     * @param response
+     * @param options
+     */
+    populateNewCourse: function(model, response, options) {
+      var self = this;
+
+      var contentObjectModel = new ProjectContentObjectModel();
+      contentObjectModel.save({
+        title: 'Page Title',
+        displayTitle: 'Page Title',
+        _type: 'page',
+        _courseId: model.get('_id'),
+        _parentId: model.get('_id')
+      },{
+        error: _.bind(this.onSaveError, self),
+        success: function() {
+          var articleModel = new EditorArticleModel();
+          articleModel.save({
+            _courseId: model.get('_id'),
+            _parentId: contentObjectModel.get('_id'),
+            _type: 'article'
+          }, {
+            error: _.bind(self.onSaveError, self),
+            success: function() {
+              var blockModel = new EditorBlockModel();
+              blockModel.save({
+                _courseId: model.get('_id'),
+                _parentId: articleModel.get('_id'),
+                _type: 'block',
+                layoutOptions: [
+                  {type: 'left', name: 'app.layoutleft', pasteZoneRenderOrder: 2},
+                  {type: 'full', name: 'app.layoutfull', pasteZoneRenderOrder: 1},
+                  {type: 'right', name: 'app.layoutright', pasteZoneRenderOrder: 3}
+                ]
+              }, {
+                error: _.bind(self.onSaveError, self),
+                success: function() {
+                  // Store the component types
+                  var componentTypes = new EditorCollection(null, {
+                    model: EditorComponentTypeModel,
+                    url: '/api/componenttype',
+                    _type: 'componentTypes'
+                  });
+
+                  componentTypes.fetch({
+                    error: _.bind(self.onSaveError, self),
+                    success: function() {
+                      var componentModel = new EditorComponentModel();
+                      componentModel.save({
+                        _courseId: model.get('_id'),
+                        _parentId: blockModel.get('_id'),
+                        body: 'This is a text component',
```

Comment:
It would ne nice if we can make this localizable ... or maybe use "Lorem ipsum ..." instead.